content/cookbook/wallets/sign-message.md

```diff
@@ -8,22 +8,44 @@ The primary function of a keypair is to sign messages and enable verification of
 the signature. Verification of a signature allows the recipient to be sure that
 the data was signed by the owner of a specific private key.
 
-To do so, we can use the [TweetNaCl](https://www.npmjs.com/package/tweetnacl)
-crypto library:
+For Solana Web3.js v1, we can use the
+[TweetNaCl](https://www.npmjs.com/package/tweetnacl) crypto library:
 
-```typescript filename="sign-message.ts"
+<Tabs groupId="language" items={['web3.js v2', 'web3.js v1']}>
+
+<Tab value="web3.js v2">
+
+```typescript
+import { generateKeyPair, signBytes, verifySignature } from "@solana/keys";
```

Comment:
Please update this to use the `@solana/web3.js` package for simplicilty.

You could add a note above the code block that explains that devs could alternatively get these imports from the specific packages. Which I think would be useful to point out.

If you do add that, please add it as a markdown quoteblock above the snippet via `> words here`. This will auto get pretty ui on the site via our `Callouts` component